src/serialize.js

```diff
@@ -38,6 +39,10 @@ function handleProperty(clone, key, value, valuePath, serializationSymbol, assig
         }
     } else if (value.constructor === Date) {
         assignments.push(valuePath + '=new Date(' + value.getTime() + ')');
+    } else if (value.constructor === URL) {
+        assignments.push(valuePath + '=new URL("' + value.href + '")');
```

Comment:
I think this `href` will need to be escaped.

Reply:
It appears that `"` is already url-encoded, should we replace any instances with `\"` to be sure?

Reply:
Good point. I think it should be fine then.